stub-environment/roles/BCPC-Hadoop-Worker.json

```diff
@@ -13,8 +13,9 @@
     "recipe[bach_krb5::krb5_client]",
     "recipe[hdfsdu::create_user]",
     "recipe[bcpc-hadoop::bach_backup_wrapper]",
-    "recipe[bcpc-hadoop::configs]",
+    "recipe[bcpc::pam]",
```

Comment:
Ron asks why did configs move?

Since there's no longer any dependency on pam in configs, we can move configs to be closer to the Hadoop components which use it.